docs/src/whatsnew/latest.rst

```diff
@@ -111,4 +113,5 @@ This document explains the changes made to Iris for this release
 .. _Pull Request Labeler GitHub action: https://github.com/actions/labeler
 .. _NPY002: https://docs.astral.sh/ruff/rules/numpy-legacy-random/
 .. _numpydoc validation: https://numpydoc.readthedocs.io/en/latest/validation.html#
-.. _Dask version 2024.2.1: https://docs.dask.org/en/stable/changelog.html#v2024-2-1
\ No newline at end of file
+.. _Dask version 2024.2.1: https://docs.dask.org/en/stable/changelog.html#v2024-2-1
+.. _@jfrost-mo: https://github.com/jfrost-mo
```

Comment:
@jfrost-mo Soz, could you move this to line `+106` in the section above, thanks.

We keep the contributor and asset URLs separate.

Reply:
Then we're good to go 🚀